packages/daemon/test/service/pipeline.test.ts

```diff
@@ -1,49 +1,76 @@
-import { app, assert } from 'midway-mock/bootstrap';
+import { app } from 'midway-mock/bootstrap';
 import { PipelineService } from '../../src/service/pipeline';
 
+const mockPipeline = {
+  id: 'mockId',
+  name: 'mockName',
+  dataCollectId: 'dataCollect',
+  dataCollect: 'dataCollect',
+  dataCollectParams: '{}',
+  dataAccessId: 'dataAccessId',
+  dataAccess: 'dataAccess',
+  dataAccessParams: 'dataAccess',
+  dataProcessId: 'dataProcess',
+  dataProcess: 'dataProcess',
+  dataProcessParams: 'dataProcess',
+  datasetProcessId: 'dataProcess',
+  datasetProcess: 'dataProcess',
+  datasetProcessParams: 'dataProcess',
+  modelDefineId: 'modelDefine',
+  modelDefine: 'modelDefine',
+  modelDefineParams: 'modelDefine',
+  modelLoadId: 'modelLoadId',
+  modelLoad: 'modelLoad',
+  modelLoadParams: '{}',
+  modelTrainId: 'modelTrain',
+  modelTrain: 'modelTrain',
+  modelTrainParams: 'modelTrain',
+  modelEvaluateId: 'modelEval',
+  modelEvaluate: 'modelEval',
+  modelEvaluateParams: 'modelEval'
+};
+
 describe('test the pipeline service', () => {
-  it('#create pipeline and get created pipeline', async () => {
+  beforeEach(async () => {
+    const pipeline: PipelineService = await app.applicationContext.getAsync<PipelineService>('pipelineService');
+    await pipeline.removeJobs();
+    await pipeline.removePipelines();
+  })
+  it('#create job', async () => {
     const pipeline: PipelineService = await app.applicationContext.getAsync<PipelineService>('pipelineService');
-    const obj = await pipeline.createPipeline({
-      dataCollect: 'dataCollect',
-      dataAccess: 'dataAccess',
-      dataProcess: 'dataProcess',
-      modelDefine: 'modelDefine',
-      modelTrain: 'modelTrain',
-      modelEvaluate: 'modelEval'
-    });
+    const obj = await pipeline.createPipeline(mockPipeline);
+    const job = await pipeline.createJob(obj.id);
+    const jobx = await pipeline.getJobById(job.id);
+    console.log(jobx);
+  });
+  // it('#create pipeline and get created pipeline', async () => {
```

Comment:
How about using `it.skip`?

Reply:
This should not be commented out, reverted.